nsv13/code/modules/overmap/knpc.dm

```diff
@@ -1,30 +1,44 @@
-#define AI_TRAIT_BRAWLER 1 //The "marines". These guys will attempt to stand and fight
-#define AI_TRAIT_SUPPORT 2 //The "medics", civvies, etc. These lads will call for backup
+///The "marines". These guys will attempt to stand and fight
+#define AI_TRAIT_BRAWLER 1
```

Comment:
the vscode extensions don't link defines to anything, so no need to worry about fixing their comments